app/services/idv/steps/inherited_proofing/agreement_step.rb

```diff
@@ -14,6 +16,19 @@ def form_submit
         def consent_form_params
           params.require(:inherited_proofing).permit(:ial2_consent_given)
         end
+
+        def enqueue_job
+          doc_capture_session = create_document_capture_session(
+            inherited_proofing_verify_step_document_capture_session_uuid_key,
```

Comment:
Surprised nothing is flagged with that dangling comma?

Reply:
Rubocop prefers the dangling comma
```Style/TrailingCommaInArguments: Put a comma after the last parameter of a multiline method call.```